src/components/Card/StyledCard.js

```diff
@@ -64,14 +64,14 @@ const buildCardMeta = (type, response) => {
       lessonCount: response.lesson_count
     },
     'lesson': {
-      langImg: response.tech_logo_http_url,
+      langImg: response.icon_url,
```

Comment:
Was this property changed? If so I need to change it too since I'm using `tech_logo_http_url` in the Instructor Center